lib/ansible/module_utils/azure_rm_common_rest.py

```diff
@@ -66,5 +78,25 @@ def query(self, url, method, query_parameters, header_parameters, body, expected
             exp = CloudError(response)
             exp.request_id = response.headers.get('x-ms-request-id')
             raise exp
+        elif response.status_code == 202:
+            def get_long_running_output(response):
+                return response
+            poller = LROPoller(self._client, ClientRawResponse(None, response), get_long_running_output, ARMPolling(30, **operation_config))
```

Comment:
1.  can you make 30 configurable?
2.   set timeout on poll?

Reply:
yes, I will add appropriate params in the module